docs/current/sdk/nodejs/guides/620941-github-google-cloud.md

```diff
@@ -5,6 +5,10 @@ displayed_sidebar: "current"
 
 # Use Dagger with GitHub Actions and Google Cloud
 
+:::note
+[Watch a live demo](https://youtu.be/-pKmv0VDJBg) of this tutorial in the Dagger Community Call (12 Jan 2023). For more demos, [join the next Dagger Community Call](https://dagger.io/events).
```

Comment:
Minor changes @mircubed, lmk what you think. Also, should we include the exact timestamp in the Youtube link so a user starts off at the right point?